torch/distributed/_tensor/ops/pointwise_ops.py

```diff
@@ -408,6 +408,20 @@
 def pointwise_strategy(
     mesh: DeviceMesh, op_schema: OpSchema, linearity: bool = False
 ) -> OpStrategy:
+    """
+    Pointwise operators can follow the strategy of its first argument
```

Comment:
hmmm I think only `list_pointwise_strategy` follows the sharding strategy of the first argument. `pointwise` strategy did more things: It could follow other operands if other operands sharding are more fine grained. So this comment might not be accurate, let's maybe add this comment to only list_pointwise_strategy